### PR TITLE
add followlike as easy

### PR DESCRIPTION
**Pull request by KienVu1504**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -7316,6 +7316,16 @@
         ]
     },
 
+    {
+        "name": "Followlike",
+        "url": "https://www.followlike.net/settings.php",
+        "difficulty": "easy",
+        "notes": "On Delete Sites or Account section, click the red 'Delete account'.",
```
Comment (tupaschoal) [inline _data/sites.json:7323]:
```suggestion
        "notes": "On Delete Sites of Account section, click the red 'Delete account'.",
```
Should this 'or' have been an 'of'? Sentence reads a bit strangely

Comment (KienVu1504) [reply _data/sites.json:7323]: it actually `Delete Sites or Account`

image

Comment (tupaschoal) [reply _data/sites.json]: Ahhh, I see, thanks !

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**